test/unit/mdc-chips/mdc-chip.test.js

```diff
@@ -120,6 +120,18 @@ test('#adapter.notifyInteraction emits ' +
   td.verify(handler(td.matchers.anything()));
 });
 
+test('#adapter.notifyTrailingIconInteraction emits ' +
+  `${MDCChipFoundation.strings.INTERACTION_EVENT}`, () => {
```

Comment:
Nit: You could omit the template string syntax here since it's already concatenated anyway